pages/guides/_guides.html

```diff
@@ -0,0 +1,106 @@
+                <img class="btn" type="radio" name="options" src="/assets/img/logos-technologies/azure_logo.png" alt="azure logo">
+            </div>
+        </div>
+    </div>
+<div class="guide-listing">
+    <div class="col-md-2 categories">
+        {% for category in site.categories %}
+        {% capture category_name %}{{ category | first }}{% endcapture %}
+        <ul>
+            <li>
+                <label><b><a href="#{{category_name | slugify}}">{{category_name}}</a></b></label>
+            </li>
+        </ul>
+        {% endfor %}
+    </div>
+    <div class="col-md-10">
+            {% for category in site.categories %}
+            {% capture category_name %}{{ category | first }}{% endcapture %}
+            <div id="#{{ category_name | slugize }}"></div>   
+                <h3 class="category-head">{{ category_name }}</h3>
+                <a name="{{ category_name | slugize }}"></a>
+                {% for post in site.categories[category_name] %}
+                <div class="guide-card card-shadow">
+                    <div class="card">
+                        <div class="row">
+                            <div class="col-md-2">
+                                <div class="card-img-top img-fluid text-center">
+                                    <img src="{{ post.image }}" alt="card image">
+                                </div>
+                            </div>
+                            <a href="{{ post.url }}">
+                                <div class="col-md-10">
+                                    <div class="card-body">
+                                        <h5 class="card-title"><b>{{ post.title }}</b></h5>
+                                        <p class="card-text">{{ post.excerpt | truncate:200 "....." }}</p>
+                                    </div>
+                                </div>
+                            </a>
+                        </div>    
+                    </div>
+                </div>
+                {% endfor %}
+            {% endfor %}
+        <div class="row newsletter">
+            <div class="col-md-9 col-md-offset-1 text-center">
+                <img src="/assets/img/guides/img-newsletter.png" alt="newsletter_image" /><br />
+                <label>Newsletter</label>
+                <p>Subscribe to the Gruntwork Newsletter to get notified when updates and new guides are released. No spam. Ever.</p>
+                <form action="#" class="col-xs-9 col-xs-offset-1">
+                    <div class="input-group">
+                        <input type="text" class="form-control" placeholder="Your Email Address">
+                        <div class="input-group-btn" >
+                            <div class="btn-group" role="group"> 
+                                <button type="button" class="btn btn-primary" >
+                                    Subscribe
```

Comment:
Is this implemented? Note that we do have a newsletter form at https://gruntwork.io/newsletter/.

Reply:
I only styled it as given in the mockup. Will implement form submission with the search and filtering